### ISSUE TITLE
nginx task failure

### ISSUE THREAD

**Issue by jtbates** [jcalazan/ansible-django-stack#76]
I'm running into this error when I run `vagrant up`:

```
TASK [nginx : Install Nginx] ***************************************************
task path: ansible-django-stack/roles/nginx/tasks/main.yml:3
fatal: [default]: FAILED! => {
  "cache_update_time": 1495215685,
  "cache_updated": true,
  "changed": false,
  "failed": true,
  "msg": "'\/usr\/bin\/apt-get -y -o \"Dpkg::Options::=--force-confdef\" -o \"Dpkg::Options::=--force-confold\"     install 'nginx'' failed: debconf: apt-extracttemplates failed: No such file or directory\nFATAL -> Failed to fork.\n",
  "stderr": "debconf: apt-extracttemplates failed: No such file or directory\nFATAL -> Failed to fork.\n",
  "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\nThe following packages were automatically installed and are no longer required:\n  comerr-dev krb5-multidev libgssrpc4 libkadm5clnt-mit9 libkadm5srv-mit9\n  libkdb5-7\nUse 'apt-get autoremove' to remove them.\nThe following extra packages will be installed:\n  nginx-common nginx-core\nSuggested packages:\n  fcgiwrap nginx-doc\nThe following NEW packages will be installed:\n  nginx nginx-common nginx-core\n0 upgraded, 3 newly installed, 0 to remove and 9 not upgraded.\nNeed to get 349 kB of archives.\nAfter this operation, 1300 kB of additional disk space will be used.\nGet:1 http:\/\/archive.ubuntu.com\/ubuntu\/ trusty-updates\/main nginx-common all 1.4.6-1ubuntu3.7 [19.0 kB]\nGet:2 http:\/\/archive.ubuntu.com\/ubuntu\/ trusty-updates\/main nginx-core amd64 1.4.6-1ubuntu3.7 [325 kB]\nGet:3 http:\/\/archive.ubuntu.com\/ubuntu\/ trusty-updates\/main nginx all 1.4.6-1ubuntu3.7 [5352 B]\nFetched 349 kB in 1s (248 kB\/s)\n",
  "stdout_lines": [
    "Reading package lists...",
    "Building dependency tree...",
    "Reading state information...",
    "The following packages were automatically installed and are no longer required:",
    "  comerr-dev krb5-multidev libgssrpc4 libkadm5clnt-mit9 libkadm5srv-mit9",
    "  libkdb5-7",
    "Use 'apt-get autoremove' to remove them.",
    "The following extra packages will be installed:",
    "  nginx-common nginx-core",
    "Suggested packages:",
    "  fcgiwrap nginx-doc",
    "The following NEW packages will be installed:",
    "  nginx nginx-common nginx-core",
    "0 upgraded, 3 newly installed, 0 to remove and 9 not upgraded.",
    "Need to get 349 kB of archives.",
    "After this operation, 1300 kB of additional disk space will be used.",
    "Get:1 http:\/\/archive.ubuntu.com\/ubuntu\/ trusty-updates\/main nginx-common all 1.4.6-1ubuntu3.7 [19.0 kB]",
    "Get:2 http:\/\/archive.ubuntu.com\/ubuntu\/ trusty-updates\/main nginx-core amd64 1.4.6-1ubuntu3.7 [325 kB]",
    "Get:3 http:\/\/archive.ubuntu.com\/ubuntu\/ trusty-updates\/main nginx all 1.4.6-1ubuntu3.7 [5352 B]",
    "Fetched 349 kB in 1s (248 kB\/s)"
  ]
}
```

**Comment by dpward** [comment]
This was caused by the VM running out of memory (`FATAL -> Failed to fork.`) It should be fixed now with commit 4bcbb78 which increases the memory allocation.

**Comment by jtbates** [comment]
That did fix it. Thanks!
